easybuild/easyconfigs/r/RMBlast/RMBlast-2.2.28-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'RMBlast'
+version = '2.2.28'
+versionsuffix = '-Python-2.7.11'
+
+homepage = 'http://www.repeatmasker.org/RMBlast.html'
+description = """RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between
+ this distribution and the NCBI distribution is the addition of a new program 'rmblastn' for use with RepeatMasker and
+ RepeatModeler."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/rmblast/%(version)s']
+sources = ['ncbi-rmblastn-%(version)s-src.tar.gz']
+checksums = [('md5', 'fb5f4e2e02ffcb1b17af2e9f206c5c22')]
+
+#    ('BLAST+', '2.3.0', versionsuffix),
```

Comment:
@boegel I did think the same, however in the release notes they say that: 
"RMBlast 2.2.27+ First integrated release of NCBI BLAST+ toolkit and RMBlast."
And they also argue that both installations are needed for precompiled (not from source) installations:
"If you choose to use the pre-compiled binaries you will need both for use with RepeatMasker/RepeatModeler."
If you think that I misunderstood these lines, then I will include the BLAST+ dependency.